test/OpSizeOf.spvasm

```diff
@@ -0,0 +1,33 @@
+; REQUIRES: spirv-as
+; RUN: spirv-as --target-env spv1.1 -o %t.spv %s
+; TODO: reenable spirv-val when OpSizeOf is supported.
+; R/U/N: spirv-val %t.spv
```

Comment:
Is there already an issue in SPIRV-Tools?

Reply:
Yes, spirv-val doesn't know much about `OpSizeOf` yet.  I'm preparing a PR to fix that.

Reply:
Just an issue reported to SPIR-V Tolls would have been good enough for now, but that's great your fix is already in progress :)

Reply:
For reference, the fix is here: https://github.com/KhronosGroup/SPIRV-Tools/pull/5879
(but even if that lands soon, we'll need to wait for the next SPIRV-Tools release before we can re-enable this check in CI.)